README.md

```diff
@@ -140,16 +141,22 @@ To enable a different DNS provider only when connected to the VPN you can add th
 
 ### HowTo become an administrator
 
-To add/revoke certificates, you must be allowed to decrypt `./cert/pki/private/ca.key.enc`.
-This file is encrypted with [sops](https://github.com/mozilla/sops) and your public gpg key must be added to `./.sops.yaml` by an existing administrator.
+To add/revoke certificates, you must be allowed to decrypt the sensitive encrypted files such as `./cert/pki/private/ca.key.enc`.
```

Comment:
```suggestion
To add/revoke certificates, you must be allowed to decrypt sensitive files such as `./cert/pki/private/ca.key.enc`.
```
(Removed "encrypted" to avoid too much repetition)

Reply:
👍 I hesitated to do the same. Make sense